docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.9'
+
+services:
+
+  db:
+    image: postgres:14-alpine
+    container_name: dbCont
+    restart: always
+    # use different exposed port from "5432" to anything else because "5432" is desired for the pgAdmin
+    ports:
+      - "5555:5432"
+
+    # change the volume place everytime you -up- the same docker compose
+    # or delete the volume everytime you -up- it
+    volumes:
+      - ~/apps/postgres:/var/lib/postgresql/data3
```

Comment:
Also try to change/delete the old volume of the db because it holds old volume from old containers even though the container stopped.

Reply:
This can be easily moved , when stopping the compose run 
'docker-compose down -v' this deletes the volumes I guess

Reply:
I didn't know it, but it's fine and working effectively